hack/make/binary

```diff
@@ -62,3 +62,18 @@ echo "Created binary: $DEST/$BINARY_FULLNAME"
 ln -sf "$BINARY_FULLNAME" "$DEST/docker$BINARY_EXTENSION"
 
 hash_files "$DEST/$BINARY_FULLNAME"
+
+# Add nested executables to bundle dir so we have complete set of
+# them available, but only if the native OS/ARCH is the same as the
+# OS/ARCH of the build target
+if [ "$(go env GOOS)/$(go env GOARCH)" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then
+	echo "Copying nested executables into $DEST"
+	(set -x
+	if [ -x /usr/local/bin/runc ]; then
+		cp /usr/local/bin/runc $DEST/
```

Comment:
ah i seeeee, ok then LGTM

On Mon, Mar 21, 2016 at 6:06 PM, Michael Crosby <notifications@github.com>
wrote:

> In hack/make/binary
> <https://github.com/docker/docker/pull/21341#discussion_r56923827>:
>
> > @@ -62,3 +62,18 @@ echo "Created binary: $DEST/$BINARY_FULLNAME"
> >  ln -sf "$BINARY_FULLNAME" "$DEST/docker$BINARY_EXTENSION"
> >
> >  hash_files "$DEST/$BINARY_FULLNAME"
> > +
> > +# Add nested executables to bundle dir so we have complete set of
> > +# them available, but only if the native OS/ARCH is the same as the
> > +# OS/ARCH of the build target
> > +if [ "$(go env GOOS)/$(go env GOARCH)" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then
> > +	echo "Copying nested executables into $DEST"
> > +	(set -x
> > +	if [ -x /usr/local/bin/runc ]; then
> > +		cp /usr/local/bin/runc $DEST/
>
> @jfrazelle <https://github.com/jfrazelle> it's also so that we can have
> these downloads available on master.dockerproject.org so that people can
> get everything they need to run docker.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/21341/files/1bf5eb20e53b7e242792fcbe399cb997b6a2ba4b#r56923827>
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>